[DNM] Renames industries following content review

1pretz1 opened the pull request:
Updates industry names after a [content review][1]. We need to keep the values the same as that is the only identifier we have to link licence documents to their tags (located in Licence Finder). A separate piece of work is currently [implementing the industry tagging migration][2].

We will come up with a cleaner solution to rename and potentially merge industries after usability testing, where we'll be more confident in the requirements, e.g the tagging structure. We want to merge this now as deploying a branch for usability  testing is fragile. 

[1]: https://docs.google.com/spreadsheets/d/1foGRMDpe67spHrXkbz_TuxcAwrZYCBdlbZPHDC-OSZs/edit#gid=741170259
[2]: https://trello.com/c/DY5YCALU/1732-tag-specialist-publisher-licences-with-imported-sectors

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

Follow [these steps](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) if you are doing a Rails upgrade.

1pretz1 committed:
Renames industries following content review

Source:
https://docs.google.com/spreadsheets/d/1foGRMDpe67spHrXkbz_TuxcAwrZYCBdlbZPHDC-OSZs/edit#gid=741170259

edwardkerry reviewed (approved):
Looks good to me (with noted caveat that it creates a discrepancy between labels and values)

1pretz1 committed:
Sort industry list alphabetically

Generated from the below query:

```
schema_file_path = Rails.root.join("lib/documents/schemas/licence_transactions.json")
json_blob = File.new(schema_file_path).read
schema = JSON.parse(json_blob)
licence_transaction_industry = schema["facets"].select { |facet| facet["key"] == "licence_transaction_industry" }
sorted_list = licence_transaction_industry.first["allowed_values"].sort { |a, b| a["label"] <=> b["label"] }

licence_transaction_industry.first["allowed_values"] = sorted_list

File.write(schema_file_path, JSON.pretty_generate(schema, { object_nl: "\n" }))
```

KludgeKML committed:
add summary text for licence_transactions page

KludgeKML committed:
Change Industry field name to Business or activity

KludgeKML committed:
Remove final line of summary
